features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/root/CreateRoomRootStateProvider.kt

```diff
@@ -27,5 +30,7 @@ open class CreateRoomRootStateProvider : PreviewParameterProvider<CreateRoomRoot
 }
 
 fun aCreateRoomRootState() = CreateRoomRootState(
```

Comment:
Maybe we we should also have state cases with:
- a search query field and and empty results
- an empty search query and empty results

Reply:
And maybe we should also move the `isSearchActive` inside the `CreateRoomRootState` to cover the different UI in the screenshots tests?

Reply:
Yes, aCreateRoomRootState() should return the "default" state, and then we can copy it (in the provider) to create more states.

Reply:
done https://github.com/vector-im/element-x-android/pull/211/commits/9c0164e9bd7317495b8253b9be8fcc56ba90256d